pkg/addon/addon.go

```diff
@@ -455,7 +503,10 @@ func cutPathUntil(path []string, end string) ([]string, error) {
 }
 
 // RenderApplication render a K8s application
-func RenderApplication(addon *types.Addon, args map[string]interface{}) (*v1beta1.Application, []*unstructured.Unstructured, error) {
+func RenderApplication(addon *types.Addon, args map[string]interface{}) (*v1beta1.Application, []*unstructured.Unstructured, []*unstructured.Unstructured, error) {
```

Comment:
We should split this function or rename it. `RenderApplicatoin` returns 3 objects(even include two array!) and one error, that's not unreasonable.